src/utils/types/user.ts

```diff
@@ -6,6 +6,7 @@ interface IUser {
     consentID?: string;
     email: string;
     userIdentifier?: string;
```

Comment:
Just out of curiosity, in the user object, there are three identifiers:

_id --> PDIs user identifier
userId --> Our internal user identifier
userIdentifier? --> Is this for... the consumer I guess? 

I don't recall ever needing to use the 'userIdentifier' for anything. Just wondering if it's necessary to include in the json returned from creating a new user as it does (did) cause (me a lot) confusion. I must have spent at least an hour trying to make sure there was nothing I could use it for. 

Just a suggestion! Great work!

Reply:
You don't need the userIdentifier, it is what is stored automatically after registration to the consent service, (see [documentation ](https://github.com/Prometheus-X-association/dataspace-connector/blob/main/docs/USER_MANAGEMENT.md) on that). The connector handles the matching of the right user to the right user identifier when communicating with the consent service.

It can be useful if you wish to implement solutions for your users regarding consent management for direct application <-> consent service interaction.